packages/quantic/force-app/main/default/lwc/sampleApp/sampleApp.html

```diff
@@ -1,30 +1,28 @@
 <template>
     <div onregisterresulttemplates={handleResultTemplateRegistration}>
-        <c-search-interface sample>
```

Comment:
Without a parent `search-interface` component, how would one configure parameters that affect the entire interface, such as locale?

Also, any thoughts around managing multiple interfaces? For example, if there was a parent component wrapping everything, would it be possible to pass down a string id, say corresponding to the engine to use on the window object?
```
window.coveoHeadless = {
    '1': engine1,
    '2: engine2
}

<c-search-interface engineId="1">
    ...
</c-search-interface>
```
I think a parent component could still be helpful to manage the complexity of powering multiple interfaces

Reply:
It's in the works already on a local branch of mine. The solution I have in mind is very similar to what you propose